File: App/Models/DomainModels/Results.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace App.Models.DomainModels
{
    public class Results : iUser
    {
        public string cristinID { get; set; }
        public string firstName { get; set; }
        public string lastName { get; set; }
        public Affiliation affiliation { get; set; }
    }
}

File: App/Ng5/src/app/searchresults/results.ts
import { Affiliation } from './affiliation';

export class Results {
  cristinID: string;
  firstName: string;
  lastName: string;
  affiliation: Affiliation;

  constructor(cristinID: string, firstName: string, lastName: string, affiliation: Affiliation) {
    this.cristinID = cristinID;
    this.firstName = firstName;
    this.lastName = lastName;
    this.affiliation = affiliation;
  }
}
